README.md

```diff
@@ -82,13 +91,25 @@ Usage of ./endlessh-go
   -prometheus_port string
         The port for prometheus (default "2112")
   -stderrthreshold value
-        logs at or above this threshold go to stderr (default 2)
+        Logs at or above this threshold go to stderr (default 2)
   -v value
-        log level for V logs
+        Log level for V logs
   -vmodule value
-        comma-separated list of pattern=N settings for file-filtered logging
+        Comma-separated list of pattern=N settings for file-filtered logging
 ```
 
+
+### Configuration File
+
+Alternatively, you can define a yaml configuration file to configure Endlessh-go. By default, no configuration file is defined.
+
+If you wish to define a configuration file, you can use the ``-config_file /path/to/config_file.yml`` command. This will overwrite the default values.
+
+You will find an exemple of a configuration file in the [exemple section](exemple/config_file/endlessh-go.yml)
+
+**Please note that if both a configuration file and CLI arguments define the same options, the CLI arguments will take precedence and overwrite the configuration file.**
```

Comment:
May I ask why do you select this precedence?

Reply:
Config file can be shared easily. There you can define most of your configuration, it's like your personnal default config. If you need a small change (for testing purpose for exemple) you can add cli args. That is what i was thinking while writing this.

Reply:
I think precedence should be :
Command line.
Config file (declared in the command line)
Environment vars

Reply:
I would say Environment vars should be the highest. That allows you put the same setup into different envs.

I am not sure whether we can customize log via config file at this moment, especially in case of CLI > config file.

Reply:
The question is not so simple. Will environment variables be used in contexts other than containers? 
Should the consideration of environment variables depend on a CLI argument "--use-env-vars"?

To my point of view, Env vars should neither overwrite CLI args neither config file declared with CLI args.